src/spells2.cc

```diff
@@ -6227,9 +6123,6 @@ bool_ passwall(int dir, bool_ safe)
 	/* Update the monsters */
 	p_ptr->update |= (PU_DISTANCE);
 
-	/* Redraw trap detection status */
-	p_ptr->redraw |= (PR_FRAME);
```

Comment:
I'm not sure this should be removed, i.e. I think the comment may be wrong. (Regardless it's harmless to leave it.)

This is also one of those things I'd like to eventually fix. All this fretting about exactly which bits of the screen to update is absolutely pointless on modern computers.